joplin/templates/wagtailadmin/shared/field.html

```diff
@@ -27,5 +27,6 @@
     </p>
 {% endif %}
 {% endcomment %}
+
```

Comment:
was this intentional?

Reply:
There is an endcomment right after this line, for some reason it isn't showing up in the diff?